build.gradle

```diff
@@ -38,6 +38,10 @@ allprojects {
     apply plugin: 'idea'
     apply plugin: 'eclipse'
 
+    project.pluginManager.withPlugin('java-base') {
```

Comment:
can we add 
```
        tasks.withType(JavaCompile) {
            options.encoding = "UTF-8"
            options.compilerArgs << '-Xlint:unchecked'
        }
```
here and remove the redundant code block from the `subprojects` configuration?

Reply:
That will work yes, then everything is a bit more together